apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -450,6 +451,19 @@ public function set_block_template( $editor_settings ) {
 	public function is_full_site_page() {
 		return 'page' === get_post_type();
 	}
+
+	/**
+	 * Add fse-enabled class to body so we can target css only if plugin enabled.
+	 *
+	 * @param array $classes classes to be applied to body.
+	 * @return array classes to be applied to body.
+	 */
+	public function add_fse_body_class( $classes ) {
+		if ( class_exists( 'A8C\FSE\WP_Template' ) ) {
```

Comment:
Since we're already in the plugin, is this check is just trivially true? (I'm not sure, maybe it's not)

Reply:
Doh! copy paste coding - will fix.